pubspec.yaml

```diff
@@ -3,24 +3,18 @@ description: High-performance asynchronous interface for SQLite on Dart and Flut
 version: 0.7.0-alpha.4
 repository: https://github.com/powersync-ja/sqlite_async.dart
 environment:
-  sdk: ">=3.2.0 <4.0.0"
+  sdk: ">=3.3.0 <4.0.0"
 
 dependencies:
-  drift: ^2.15.0
-  # In order to compile custom web worker
-  # drift:
-  #   git:
-  #     url: https://github.com/powersync-ja/drift.git
-  #     ref: test # branch name
-  #     path: drift
-  sqlite3: "^2.3.0"
-  js: ^0.6.3
+  sqlite3_web: ^0.1.2-wip
+  sqlite3: ^2.4.3
   async: ^2.10.0
   collection: ^1.17.0
   mutex: ^3.1.0
   meta: ^1.10.0
 
 dev_dependencies:
+  js: ^0.6.7
```

Comment:
I might have missed it but what is this dep for?

Reply:
It's only used [here](https://github.com/powersync-ja/sqlite_async.dart/blob/b77b55a71aa34c65ef18bf72541415fe27aa2f84/test/utils/web_test_utils.dart#L9) for tests. It's basically used for mapping to the JavaScript environment in web. Previously this was a dependency of `sqlite.dart`, but they removed the dependency there.